grails-app/views/layouts/unguarded_component.gsp

```diff
@@ -0,0 +1,119 @@
+    <script>
+        if (location.protocol !== "https:") location.protocol = "https:";
+        function onSignIn(googleUser) {
+            // show spinner while the page is re-loading
+            document.getElementById("login_spinner").setAttribute("class", "visible");
+            var profile = googleUser.getBasicProfile();
+            var token = googleUser.getAuthResponse().id_token;
+            var auth2 = gapi.auth2.getAuthInstance();
+            auth2.then(function() {
+                var xhttp = new XMLHttpRequest();
+                <auth:isNotAuthenticated>
+                xhttp.onreadystatechange = function() {
+                    if (this.readyState == 4 && this.status == 200) {
+                        location.reload();
+                    }
+                };
+                </auth:isNotAuthenticated>
+                var url = "${raw(createLink(controller: "auth", action: "authUser"))}";
+                var postData = "token=" + token;
+                xhttp.open("POST", url, true);
+                xhttp.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
+                xhttp.send(postData);
+            });
+        }
+        function signOut() {
+            var auth2 = gapi.auth2.getAuthInstance();
+            auth2.signOut().then(function (){
+                var xhttp = new XMLHttpRequest();
+                xhttp.onreadystatechange = function() {
+                    if (this.readyState == 4 && this.status == 200) {
+                        window.location = "${request.contextPath}";
+                    }
+                };
+                var url = "${raw(createLink(controller: "logout", action: "logout"))}";
+                xhttp.open("POST", url, true);
+                xhttp.send();
+            });
+        }
+    </script>
+    <g:layoutHead/>
+</head>
+<body>
+<g:render template="/base/topNav" />
+
+<auth:isNotAuthenticated>
+    <div class="container">
+     <div id="login_spinner" class="hidden">
+              <div class="alert alert-success alert-dismissable" style="display: block">
+                  Loading ... <span class="glyphicon glyphicon-refresh glyphicon-refresh-animate"></span>
+              </div>
+          </div>
+        <g:render template="/base/messages" />
+        <g:layoutBody/>
+    </div>
+</auth:isNotAuthenticated>
+
+    <auth:nonBroadSession>
+    <div class="container">
+        <div class="alert alert-danger alert-dismissable" style="display: block">
+            You must be a Broad Institute User for further access. Please sign out and log in with
+            a "broadinstitute.org" email account.
+        </div>
+        <h3>About the ORSP Portal</h3>
+        <g:render template="/index/aboutBlurb"/>
+    </div>
+</auth:nonBroadSession>
+
+<auth:broadSession>
+    <div class="container">
+        <g:render template="/base/messages" />
+        <g:layoutBody/>
+    </div>
+</auth:broadSession>
```

Comment:
The three auth blocks above can be condensed into one. Since the page doesn't show different content for different users, all three can just be:
```
<div class="container">
    <g:render template="/base/messages" />
    <g:layoutBody/>
</div>
```

Reply:
One correction to the above ... the about page link might need to be updated based on recent conversations around when to show that.